Il2CppInterop.Runtime/DelegateSupport.cs

```diff
@@ -272,7 +274,7 @@ private static void LogError(string message)
                 throw new ArgumentException(
                     $"Parameter type at {i} has mismatched native type pointers; types: {nativeType.FullName} != {managedType.FullName}");
 
-            if (nativeType.IsByRef || managedType.IsByRef)
+            if (nativeType.IsByRef && managedType.IsByRef)
```

Comment:
What's the reason for this change?

Reply:
I don't actually remember. Probably a stray change. Will revert.